core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -308,15 +320,70 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
             </div>
           </div>
 
+          <div ng-if="activeSection === LEARNER_DASHBOARD_SECTION_I18N_IDS.PLAYLIST && activeSubsection === LEARNER_DASHBOARD_SUBSECTION_I18N_IDS.EXPLORATIONS">
+            <span style="display: block; padding: 5px 7.5px;"
+                  ng-if="explorationPlaylist.length > 0">
+              <span translate="I18N_LEARNER_DASHBOARD_REARRANGE_LEARNER_PLAYLIST_MESSAGE"></span>
+            </span>
+            <span style="display: block; padding: 5px 7.5px;"
+                  ng-if="numberDeletedExplorationPlaylist !== 0">
+              <span translate="I18N_LEARNER_DASHBOARD_DELETED_EXPLORATIONS_FROM_PLAYLIST"
+                    translate-values="{numberDeleted: <[numberDeletedExplorationPlaylist]>}"
+                    translate-interpolation="messageformat"></span>
+              &#9785
+            </span>
+
+            <md-card layout="row"
+                     class="oppia-learner-dashboard-no-activity-card"
+                     ng-if="explorationPlaylist.length === 0">
+              <div>
+                <p translate="I18N_LEARNER_DASHBOARD_EMPTY_EXPLORATION_PLAYLIST"></p>
+
+                <a class="btn oppia-dashboard-intro-button oppia-transition-200"
+                   translate="I18N_ACTION_BROWSE_LESSONS"
+                   ng-href="/library"
+                   style="width: 230px; margin-top: 13px;">
+                </a>
+              </div>
+            </md-card>
+
+            <div ui-sortable="explorationPlaylistSortableOptions" ng-model="explorationPlaylist">
+              <div class="oppia-learner-playlist-tiles"
+                   ng-repeat="tile in explorationPlaylist track by $index"
+                   ng-mouseenter="showRemoveIcon=true" ng-mouseleave="showRemoveIcon=false"
+                   style="height: 145px;">
+                <i class="remove-icon fa fa-times"
+                   ng-show="showRemoveIcon"
+                   aria-hidden="true"
+                   style="color: black;" 
+                   ng-click="openRemoveActivityModal(activeSection, activeSubsection, tile)"
+                   tooltip="Remove"
+                   tooltip-placement="top"></i>
+                <exploration-summary-tile
+                                        style="height: 145px; display: block;"
```

Comment:
Move this to end of previous line, and revise the indentation accordingly. It's OK if this causes subsequent lines to go > 80 chars.

Reply:
Done!